spec/exchanges/volume_tracking/message_builder_spec.rb

```diff
@@ -51,7 +51,7 @@
                                                           source_type: 'library',
                                                           source_barcode: pacbio_library.tube.barcode,
                                                           sample_name: pacbio_library.sample_name,
-                                                          used_by_type: 'well',
+                                                          used_by_type: 'run',
```

Comment:
I haven't been involved in a lot of this work so this might have already been decided but is this right being 'run'? The used_by_barcode is a well barcode so does that need changing?

Reply:
Good question. In Avro [schema](https://redpanda.uat.psd.sanger.ac.uk/console/schema-registry/subjects/create-aliquot-in-mlwh?version=latest) it has been defined as a run as a source_type, but then the barcode field points to well. I will clarify it with @stevieing

Reply:
@BenTopping Just to let you know, I've discussed with Steve and confirmed that it should be 'run' as it makes more sense to external users.